paper-card.html

```diff
@@ -58,7 +58,7 @@
 
 <dom-module id="paper-card">
   <template>
-    <style>
+    <style include="paper-material">
```

Comment:
Also it's not getting extended, really. There's dark magic where if you're style-including a dom-module, only its styles (and not the template) are brought in :/